Sources/BraveVPN/BraveVPN.swift

```diff
@@ -300,6 +314,23 @@ public class BraveVPN {
       completion?(status)
     }
   }
+  
+  /// Attempts to reconfigure the vpn fpr protocol change
```

Comment:
Typo "fpr"

Reply:
The function is removed with migration Changes https://github.com/brave/brave-ios/pull/7125/commits/4dfc0287513c4b44498e3cd43ddcc0c6bd00946e